### PR TITLE
LPS-177405: remove unnecesarry session massages

### PR DESCRIPTION
**Pull request by kristofloneck**
Messages generated by these line are extra spams. See: LPP-48038

### CODE REVIEW

**Comment by liferay-lima** [comment]
ci:test:sf

File: modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/portlet/action/EditFileEntryMVCActionCommand.java [hunk 1]
```diff
@@ -348,15 +348,6 @@ else if (cmd.equals(Constants.PREVIEW)) {
 
 			String portletResource = ParamUtil.getString(
 				actionRequest, "portletResource");
-
-			if (Validator.isNotNull(portletResource) ||
-				cmd.equals(Constants.ADD_DYNAMIC)) {
-
-				hideDefaultSuccessMessage(actionRequest);
-
-				MultiSessionMessages.add(
```
Comment (AliciaGarciaGarcia) [inline modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/portlet/action/EditFileEntryMVCActionCommand.java:357]: If you delete all this code the sucess message will appears 2 times :


https://user-images.githubusercontent.com/47524751/223499524-40293f19-db4e-4954-b619-aad25fa7bbb7.mp4



I think that removing only this 
`MultiSessionMessages.add(...`
The issue will be solved

try both scenarios

Thanks

**Review by AliciaGarciaGarcia — CHANGES_REQUESTED** [review]

**Comment by AliciaGarciaGarcia** [comment]
Hi @kristofloneck 

Have you read my comments?

**Comment by kristofloneck** [comment]
> Hi @kristofloneck
> 
> Have you read my comments?

Yes I'm started doing it.

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

File: modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/portlet/action/EditFileEntryMVCActionCommand.java [hunk 2]
```diff
@@ -348,6 +348,10 @@ else if (cmd.equals(Constants.PREVIEW)) {
 
 			String portletResource = ParamUtil.getString(
 				actionRequest, "portletResource");
+			if (Validator.isNotNull(portletResource) ||
```
Comment (AliciaGarciaGarcia) [inline modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/portlet/action/EditFileEntryMVCActionCommand.java:351]: Hi @kristofloneck 

The code does not compile, can you recheck it

Thanks!

**Review by AliciaGarciaGarcia — CHANGES_REQUESTED** [review]

File: modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/portlet/action/EditFileEntryMVCActionCommand.java (hunk 2 above)
Comment (kristofloneck) [reply modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/portlet/action/EditFileEntryMVCActionCommand.java:351]: check it, fixed it.

**Comment by kristofloneck** [comment]
ci:test:sf

**Comment by kristofloneck** [comment]
I restart my local sf check because It skipped some stuff.

**Comment by kristofloneck** [comment]
ci:test:sf

**Comment by AliciaGarciaGarcia** [comment]
Hi @kristofloneck 

Can you confirm us that you checked the solution works for both scenarios?

Thanks

**Comment by kristofloneck** [comment]
In the other scenario only one of the toast boxes will be removed. I tried to think somekind of logics which can help to distinguish the two case but I did not find yet.

**Comment by AliciaGarciaGarcia** [comment]
thanks for checking it @kristofloneck

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Review by AliciaGarciaGarcia — APPROVED** [review]

**Comment by AliciaGarciaGarcia** [comment]
ci:reevaluate:1407401_721

**Comment by AliciaGarciaGarcia** [comment]
ci:forward

**Comment by kristofloneck** [comment]
wait where is my message?

**Comment by kristofloneck** [comment]
I wrote a message about this don't solved the issue.

**Comment by AliciaGarciaGarcia** [comment]
Sorry @kristofloneck 

I'm lost here, do you want me to close the PR on Brian and resend another PR?

**Pull request closed without merging**